webpack/assets/javascripts/react_app/components/bookmarks/index.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Dropdown, MenuItem } from 'patternfly-react';
+import SearchModal from './SearchModal';
+import Bookmark from './Bookmark';
+import * as BookmarkActions from '../../redux/actions/bookmarks';
+import DocumentationUrl from '../common/DocumentationLink';
+
+class BookmarkContainer extends React.Component {
+  componentDidMount() {
+    const { url, controller, getBookmarks } = this.props;
+
+    getBookmarks(url, controller);
+  }
+
+  onClick() {
+    if (this.props.showModal) {
+      this.props.modalClosed();
+    } else {
+      this.props.modalOpened();
+    }
+  }
+
+  render() {
+    const {
+      controller, url, showModal, modalClosed, canCreate,
+    } = this.props;
+
+    return showModal ? (
+      <SearchModal show={showModal} controller={controller} url={url} onHide={modalClosed} />
+    ) : (
+      <Dropdown pullRight id={controller}>
```

Comment:
Will be easier and cleaner to use `DropdownButton`
http://rawgit.com/sharvit/patternfly-react/storybook/dropdown-buttons/index.html?knob-Label=Danger%20Will%20Robinson%21&knob-Style=default&selectedKind=Button&selectedStory=DropdownButton&full=0&down=1&left=1&panelRight=0&downPanel=storybooks%2Fstorybook-addon-knobs

Reply:
is it merged already?

Reply:
You right, it won't work without the `title` prop :+1: